### DISCUSSION TITLE
dashboard for dependabot alerts

### DISCUSSION THREAD

**Discussion by sunithagorty-drizly** [community/community#34386] [Code Security]
Is there an existing integration or solution that can help generate a dashboard for dependabot alerts?

**Comment by naomi789** [comment]
Yes! Are you familiar with [Security Overview](https://docs.github.com/en/code-security/security-overview/about-the-security-overview)? I actually work at GitHub and I'd be really interested in getting your feedback as to whether this would meet your needs - or if not, what you'd be looking for?

**Comment by sunithagorty-drizly** [comment]
Thank you for getting back to me, Naomi! I am familiar with the security overview dashboard. That gives a current snapshot of the alerts from what I understand.
Since you are asking for feedback :-):
1. I am trying to come up with a dashboard that shows trend on how the vulnerabilities are remediated in my organization. Ability to filter on specific repos. 
2. Show a graph with age: how many X severity of vulnearibilities existed for more than a month, more than 3 months, more than a year etc. And if this configurable that will be great as each organization has different SLAs for severity remediation.
